app/controllers/application_controller.rb

```diff
@@ -12,7 +12,7 @@ def set_locale
     end
   end
 
-  def default_url_options
+  def default_url_options(options = {})
```

Comment:
Unused method argument - `options`. If it's necessary, use `_` or `_options` as an argument name to indicate that it won't be used. You can also write as `default_url_options(*)` if you want the method to accept any arguments but don't care about them.